includes/amp-helper-functions.php

```diff
@@ -842,9 +842,8 @@ function amp_get_boilerplate_stylesheets() {
 /**
  * Add generator metadata.
  *
- * @since 6.0
```

Comment:
Why is `@since 6.0` removed?

Reply:
Oh, I think this was a typo and it was supposed to originally be `0.6`

Reply:
I assumed it was a typo. Is it referring to something I'm not familiar with?